README.md

```diff
@@ -9,24 +8,28 @@ This kubectl plugin allows you to solve this issue:
 
 This command essentially wraps around `kubectl run`. It will allow you to specify a deployment name and the deployment namespace, and it will automatically grab the currently running image, resource limits/requests, config and secret mappings so that you have the required environment variables to complete your one-off task without impacting the existing deployment.
 
+### Pre-requisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [jq](https://github.com/stedolan/jq/wiki/Installation)
+
 ### Installation
 
+TODO: automated install method
```

Comment:
```suggestion
TODO: automated install method
```

Please don't place TODO's in README, we can open an issue instead.

Reply:
Sure, let's create an issue, but why don't you want to place a TODO in the README?

Personally I think helps the reader to understand that section is not completed yet and actually is a pretty common thing to do in documentation in general.

Reply:
It's generally good practice to have a single source of truth for todo's. In git and on GitHub, this is the issue system, since it is tracked.